bedrock/base/templates/macros-protocol.html

```diff
@@ -27,11 +27,14 @@
 {%- endmacro %}
 
 {# Hero: https://protocol.mozilla.org/patterns/organisms/hero.html #}
-{% macro hero(title, desc=None, class=None, include_cta=False, heading_level=2, image_url=None, include_highres_image=False, image_alt='') -%}
+{% macro hero(title, subheading, desc=None, class=None, include_cta=False, heading_level=2, subheading_level=3, image_url=None, include_highres_image=False, image_alt='') -%}
 <section class="mzp-c-hero{% if class %} {{ class }}{% endif %}{%if image_url%} mzp-has-image{% endif %}">
   <div class="mzp-l-content">
     <div class="mzp-c-hero-body">
       <h{{ heading_level }} class="mzp-c-hero-title">{{ title }}</h{{ heading_level }}>
+      {% if subheading %}
+      <h{{ subheading_level }} class="mzp-c-hero-subheading">{{ subheading }}</h{{ subheading_level }}>
```

Comment:
`mzp-c-hero-subheading` isn't a class name that exists in the protocol component, so I'd suggest maybe leaving it out for now? We could file an issue to include a sub-heading in the original component if we want to add this.

Another way to fix this without modifying the macro would be include a custom sub heading within the hero description. You can add any kind of arbitrary HTML underneath the hero heading using the macro. An example of how to do this can be [seen here](https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/templates/firefox/enterprise/index.html#L45-L53).

That said, what you have here does work - whichever way you want to fix this is fine, but I'd suggest at least removing the non-existant clas name.

Reply:
Doh, I forgot you're already doing a custom hero description (it's early I need more coffee) :)

So in this case you could do the sub heading by adding the following at line 50:

```
  <h2>{{ _('Take your passwords everywhere') }}</h2>
  <p class="mzp-c-hero-desc">{{ _('Securely access the passwords you’ve saved in Firefox from anywhere — even outside of the browser.') }}</p>
```

Reply:
We discussed this on Slack, and @Errolyn is going to file a protocol issue to incorporate a sub-heading in the hero component. We're going to keep this as-is, with the aim to bring these changes back into Protocol.